index.html

```diff
@@ -37,7 +37,7 @@
             </div>
 
             <div class="editor-info">
-                <span class="infotext">Editor: 1.56 ─ Game: v0.10.0 B1</span>
+                <span class="infotext">Editor: 1.57 ─ Game: v0.10.0 B1</span>
```

Comment:
Mentioned in Discord: Please revert this part, I keep the version in sync with upstream (Kyromyr)

Reply:
I also entered a PR in the upstream, but I can revert.